<!--The most recent publications from the group-->
<div class=publicationsmall>
    <p>
      Hobbs, E. E. M., Gloster, T. M., Pritchard, L. (2022) 'cazy_webscraper: local compilation and  
      interrogation of comprehensive CAZyme datasets'. bioRxiv (PrePrint). 
      <a href="https://www.biorxiv.org/content/10.1101/2022.12.02.518825v1">https://www.biorxiv.org/content/10.1101/2022.12.02.518825v1</a>
    </p>
    <p>
      McLellan, H., Harvey, S. E., Steinbrenner, J., Armstrong, M. R., He, Q., Clewes, R., Pritchard, L.,  
      Wang, W., Wang, S., Nussbaumer, T., Dohai, B., Luo, Q., Kumari, P., Duan, H., Roberts, A., Boevink,  
      P. C., Neumann, C., Champouret, N., Hein, I., ... Birch, P. R. J. (2022). Exploiting breakdown in  
      nonhost effector-target interactions to boost host disease resistance. Proceedings of the National  
      Academy of Sciences of the United States of America, 119(35), [e2114064119].  
      <a href="https://doi.org/10.1073/pnas.2114064119">https://doi.org/10.1073/pnas.2114064119</a>
    </p>
    <p>
      Mazloom, R., Pritchard, L., Brown, C. T., Vinatzer, B. A., & Heath, L. S. (2022). LINgroups as a  
      principled approach to compare and integrate multiple bacterial taxonomies. In Proceedings of  
      the 13th ACM International Conference on Bioinformatics, Computational Biology and Health Informatics,  
      BCB 2022: roceedings of the 13th ACM International Conference on Bioinformatics, Computational Biology  
      and Health Informatics (pp. 1-7). [55] (Proceedings of the 13th ACM International Conference on  
      Bioinformatics, Computational Biology and Health Informatics, BCB 2022). <a href="https://doi.org/10.1145/3535508.3545546">https://doi.org/10.1145/3535508.3545546</a>
    </p>
    <p>
      Feeney, M. A., Newitt, J. T., Addington, E., Algora-Gallardo, L., Allan, C., Balis, L., Birke, A. S.,  
      Casta√±o-Espriu, L., Charkoudian, L. K., Devine, R., _et al._ (2022). ActinoBase: tools and protocols  
      for researchers working on Streptomyces and other filamentous actinobacteria. Microbial Genomics, 8(7),  
      [000824]. <a href="https://doi.org/10.1099/mgen.0.000824">https://doi.org/10.1099/mgen.0.000824</a>
    </p>
    <p>
        Pritchard, L., Sharma, P., S., Mazloom, R., Piece, T., Irber, L., Harrington, B., Heath, L., Brown, T., Vinatzer, B. 
        (2022) 'genomeRxiv: a microbial whole-genome database and diagnostic marker design resource for 
        classification, identification, and data sharing', 
        <i>Microbiology Society Annual Conference 2021</i> 
        <a href="https://doi.org/10.1099/acmi.ac2021.po0165">https://doi.org/10.1099/acmi.ac2021.po0165</a>
    </p>
    <p>
        Pritchard, L. Brown, T., Harrington, B., Heath, L., Pierce-Ward, N. T., Vinatzer, B. A. 
        (2022) 'Could a focus on the "why" of taxonomy help taxonomy better respond to the needs 
        of science and society?', <i>Frontiers in Microbiology</i>, Vol. 13, 
        <a href="https://doi.org/10.3389/fmicb.2022.887310">https://doi.org/10.3389/fmicb.2022.887310</a>
    </p>
</div>
